packages/canary-integration/package.json

```diff
@@ -24,7 +24,7 @@
     "@stablelib/random": "^1.0.1",
     "@truffle/hdwallet-provider": "^2.0.0",
     "ceramic-cacao": "^0.0.16",
-    "dids": "^3.0.0-alpha.9",
+    "dids": "^3.0.0-alpha.11",
```

Comment:
@haardikk21 There is an even newer `3.0.0-alpha.12`. Could you please use it here?